src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -156,7 +163,7 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
     if (this.shapefile[0] && this.shapefile[0].documentFileName && this.shapefile[0].documentFileName.length > 0) {
       console.log('Shapefile', this.shapefile);
       const escapedName = encode(this.shapefile[0].documentFileName).replace(/\(/g, '%28').replace(/\)/g, '%29').replace(/\\/g, '_').replace(/\//g, '_').replace(/\%2F/g, '_');
-      const shapeurl = '/api/document/' + this.shapefile[0]._id + '/fetch/' + escapedName;
+      const shapeurl = this.pathAPI + '/document/' + this.shapefile[0]._id + '/fetch/' + escapedName;
```

Comment:
are you sure there will always be a shapefile._id here...

Reply:
Added a check for _id :)